features/steps/core.py

```diff
@@ -66,6 +67,12 @@ def set_config(context, config_file):
             cf.write("version: {}".format(__version__))
 
 
+@when('we open the editor and exit')
```

Comment:
I would make this test a bit more general so we can reuse it for additional tests. Something like the `run_with_input` method below. 

```
@when('we open the editor and enter')
@when('we open the editor and enter {text}')
```
And then patch the get_text_from_editor with `(context.text or text)`